ansible/roles/motis/templates/motis-update-feeds-secondary.sh

```diff
@@ -5,12 +5,14 @@
 
 echo "Copying new files…"
 
-#cp -r -p /var/lib/motis/data /var/cache/transitous/out/data
-sudo -u motis cp -r -p /var/cache/transitous/out/osm/ /var/cache/transitous/out/data/ # TODO -l ?
 
 cd /var/cache/transitous/out/
-sudo -u motis wget -N -nH --cut-dirs=1 --exclude-directories=/gtfs/data/tiles/,/gtfs/data/osr/ --mirror --no-parent -e robots=off https://api.transitous.org/gtfs/data/ || true
-sudo -u motis wget -N -e robots=off https://api.transitous.org/gtfs/config.yml || true
+sudo -u motis wget --limit-rate=30m --mirror -l 1 --no-parent --no-directories --accept gtfs.zip --accept config.yml -e robots=off https://api.transitous.org/gtfs/ || true
+#sudo -u motis wget --limit-rate=30m --mirror -l 1 --no-parent --no-directories --accept gtfs.zip -e robots=off https://api.transitous.org/gtfs/ || true
+
+sudo -u motis cp -r -p --reflink=auto /var/cache/transitous/out/osm/ /var/cache/transitous/out/data/
+sudo -u motis /opt/motis/motis_new import -c /var/cache/transitous/out/config.yml > /var/cache/transitous/motis-import.log 2>&1
```

Comment:
```suggestion
sudo -u motis /opt/motis/motis import -c /var/cache/transitous/out/config.yml > /var/cache/transitous/motis-import.log 2>&1
```
This was probably just for upgrading?